projects/sonar/src/app/_layout/admin/admin.component.html

```diff
@@ -14,52 +14,47 @@
  You should have received a copy of the GNU Affero General Public License
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
-<ngx-spinner
-  [fullScreen]="true"
-  type="ball-spin-clockwise"
-  size="medium"
-  bdColor="rgba(51,51,51,0.2)"
-  color="#000"
->
+<ngx-spinner [fullScreen]="true" type="ball-spin-clockwise" size="medium" bdColor="rgba(51,51,51,0.2)" color="#000">
 </ngx-spinner>
 <div *ngIf="ready">
   <div *ngIf="user; else errorTemplate">
     <nav class="navbar navbar-expand-sm navbar-dark bg-primary">
       <div class="container">
-        <a class="navbar-brand" routerLink=""
-          >SONAR <span class="text-secondary">administration</span></a
-        >
-        <button
-          class="navbar-toggler d-lg-none"
-          type="button"
-          aria-expanded="false"
-          (click)="isCollapsed = !isCollapsed"
-        >
+        <a class="navbar-brand" routerLink="">SONAR <span class="text-secondary">administration</span></a>
+        <button class="navbar-toggler d-lg-none" type="button" aria-expanded="false"
+          (click)="isCollapsed = !isCollapsed">
           <i class="fa fa-bars"></i>
         </button>
         <div class="collapse navbar-collapse" [collapse]="isCollapsed">
           <ul class="navbar-nav mr-auto">
-            <li class="nav-item">
-              <a class="nav-link" routerLink="/records/documents">{{ 'Records' | translate }}</a>
+            <li class="nav-item" *ngIf="user.is_admin">
+              <a class="nav-link" routerLink="/records/documents/documents" translate>Documents</a>
             </li>
-            <li class="nav-item">
-              <a class="nav-link" routerLink="/deposit/0/create"
-                >{{ 'Deposit a publication' | translate }}
+            <li class="nav-item dropdown" *ngIf="user.is_publisher" dropdown>
+              <a class="nav-link dropdown-toggle" href (click)="(false)" role="button" data-toggle="dropdown"
```

Comment:
click seems a bit weird

Reply:
I did this to have a link cursor, instead of a text cursor. But anyway I do it in CSS, it's better